src/views/Workspace/Cards/DonateCard.jsx

```diff
@@ -0,0 +1,589 @@
+  </SvgIcon>
+);
+
+const apiUrl = 'https://dev.api.forus.io/api/v1';
+
+const mocks = {
+    names: ['Chloe', 'Maximillian', 'Den', 'Bill', 'James', 'Peter'],
+    notes: ['Longfonds: flyers', 'Success allemaal'],
+};
+
+// step:  "Doneer"
+class DonateCard extends Component {
+    state = {
+        dialogOpen: false,
+        bank: '',
+        banks: [],
+        bankFetched: false,
+        amount: 25,
+        comment: '',
+        isAnonymousDonation: true,
+        isMoneyToAnotherActie: true,
+        view: 'beide',
+        submitLoading: null,
+        redirectUrl: null,
+        totalBudget: null,
+        transactionsIn: [],
+        transactionsOut: [],
+    };
+
+    componentDidMount() {
+        this.getTransactions();
+        this.getBanks();
+        this.getTotalBudget();
+    }
+
+    fetchTransactionsRequest = (fetchUrl, direction) => {
```

Comment:
All these data requests should move to separate actions and reducers

Reply:
Oh, sure, done!